auth/auth.go

```diff
@@ -65,13 +88,16 @@ type Challenge interface {
 // and required access levels for a request. Implementations can support both
 // complete denial and http authorization challenges.
 type AccessController interface {
-	// Authorized returns non-nil if the request is granted access. If one or
-	// more Access structs are provided, the requested access will be compared
-	// with what is available to the request. If the error is non-nil, access
-	// should always be denied. The error may be of type Challenge, in which
-	// case the caller may have the Challenge handle the request or choose
-	// what action to take based on the Challenge header or response status.
-	Authorized(req *http.Request, access ...Access) error
+	// Authorized returns a non-nil error if the request is granted access and
```

Comment:
An `AccessController` no longer authorizes a request; it produces an authorized context.

Should read as follows:

```
// Authorized returns a non-nil error if the context is granted access and
// ...
```